tests/functional/application-packages/Execute_WorkflowSubsetLLNL_SubsetCRIM.json

```diff
@@ -3,39 +3,48 @@
   "response": "document",
   "inputs": [
     {
-      "id": "tas",
-      "href": "https://aims3.llnl.gov/thredds/fileServer/css03_data/CMIP6/CMIP/NASA-GISS/GISS-E2-1-G/historical/r1i1p1f1/Amon/tas/gn/v20180827/tas_Amon_GISS-E2-1-G_historical_r1i1p1f1_gn_185001-190012.nc"
+      "id": "files",
+      "href": "https://boreas.ouranos.ca/twitcher/ows/proxy/thredds/dodsC/birdhouse/nrcan/nrcan_northamerica_monthly/tasmin/nrcan_northamerica_monthly_2015_tasmin.nc"
     },
     {
-      "id": "llnl_lat0",
-      "data": "33"
-    },
-    {
-      "id": "llnl_lat1",
-      "data": "60"
-    },
-    {
-      "id": "llnl_lon0",
-      "data": "-128"
+      "id": "variable",
+      "data": "tasmin"
     },
     {
-      "id": "llnl_lon1",
-      "data": "-48"
+      "id": "esgf_api_key",
+      "data": "G2LhbLn4clvTW9SHM10D0ZGPFPkEeCmm7zJU0LPJ7mTaJzrgkJ3tazcsxdCp3nl5"
```

Comment:
Good chance this shouldn't be here as a literal string
We could use a Template string, and replace with an env var.
For test on travis, the value should be pushed as an encrypted value, which is specified in .travis.yml